controllers/topics.controllers.js

```diff
@@ -0,0 +1,10 @@
+const fetchTopics = require('../models/topics.models.js');
+
+function getTopics(req,res,next) {
+    fetchTopics()
+    .then((topics) => {
+        res.status(200).send(topics)
+    })
```

Comment:
You have no catch block currently, this will need to be implemented as it is how we are going to pass errors to our error handling middleware functions.